attestation-ca/src/lib.rs

```diff
@@ -99,79 +114,17 @@ pub struct AttestationCaList {
     pub cas: BTreeMap<Base64UrlSafeData, AttestationCa>,
 }
 
-impl TryFrom<AttestationCa> for AttestationCaList {
```

Comment:
Update `webauthn-authenticator-rs/src/softtoken.rs:932:10` and `webauthn-authenticator-rs/src/softtoken.rs:1014:10` for this